Cargo.toml

```diff
@@ -23,3 +23,4 @@ rust-embed = "5.7.0"
 inkwell = { version = "0.1.0-beta.2", features = ["llvm10-0"], optional = true }
 regex = "1.5.4"
 lazy_static = "1.4.0"
+indextree = "4.3.1"
```

Comment:
why

Reply:
I'm just trying out if this idea actually carries fruit. I'm sure we can implement our own tree down the line.